ext/odbc/tests/odbc_free_result_001.phpt

```diff
@@ -11,15 +11,13 @@ include 'config.inc';
 
 $conn = odbc_connect($dsn, $user, $pass);
 
-odbc_exec($conn, 'CREATE DATABASE odbcTEST');
+odbc_exec($conn, 'CREATE TABLE free_result (TEST INT NOT NULL)');
+odbc_exec($conn, 'ALTER TABLE free_result ADD PRIMARY KEY (TEST)');
 
-odbc_exec($conn, 'CREATE TABLE FOO (TEST INT NOT NULL)');
-odbc_exec($conn, 'ALTER TABLE FOO ADD PRIMARY KEY (TEST)');
+odbc_exec($conn, 'INSERT INTO free_result VALUES (1)');
+odbc_exec($conn, 'INSERT INTO free_result VALUES (2)');
```

Comment:
We can insert the data in one request
```suggestion
odbc_exec($conn, 'INSERT INTO free_result VALUES (1), (2)');
```

Reply:
Oh, I now understand those "Could be one INSERT query" comments ^^